docs/README.md

```diff
@@ -1149,7 +1149,7 @@ export class EventName {
 }
 ```
 
-Events and [entities](#4-entities-and-reducers) are intimately related. Each event belongs to one entity through the `entityID` method, and entities represent the application's state after reducing the stream of events. Indeed, an entity is just an aggregated representation of the same data present in its events, so it is possible to rebuild entities from events at any time. Booster guarantees that all the events associated with an entity will be reduced in the same order they were stored. Take a look at this event:
+Events and [entities](#4-entities-and-reducers) are intimately related. Each event belongs to one entity identified through the `entityID` method, and entities represent the application's state after reducing the stream of events. Indeed, an entity is just an aggregated representation of the same data present in its events, so it is possible to rebuild entities from events at any time. Booster guarantees that all the events associated with an entity will be reduced in the same order they were stored. Take a look at this event:
```

Comment:
`are intimately related` -> `are closely related` 
`Each event belongs to one entity identified through the entityID method` -> `Each event belongs to one entity uniquely identified through the entityID method` this is true right? it is unique and can be identified with tis entityID

Reply:
That is right @rdoria1. It is uniquely identified through its entityID.

Great recommendations, I will work on it. Thank you :D